pkg/pipeline/common_steps_test.go

```diff
@@ -0,0 +1,359 @@
+package pipeline
+
+import (
+	"fmt"
+	"os"
+	"testing"
+	"time"
+
+	"github.com/projectsyn/lieutenant-operator/pkg/apis"
+	synv1alpha1 "github.com/projectsyn/lieutenant-operator/pkg/apis/syn/v1alpha1"
+	"github.com/stretchr/testify/assert"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/kubectl/pkg/scheme"
```

Comment:
I think this should be `"k8s.io/client-go/kubernetes/scheme"`, no?

I really hope we don't have to pull in the `k8s.io/kubectl` dependency.

Reply:
Looks like the vscode autoimporter grabbed some random dependency again.